src/components/General/DateAsWords.spec.js

```diff
@@ -0,0 +1,24 @@
+jest.mock('@/store/plugins/persistedState', () => {
+  return () => {}
+})
+
+import DateAsWords from './DateAsWords'
+import { mountWithDefaults } from '>/helpers'
+import store from '@/store'
+
+describe('DateAsWords', () => {
+  it('renders 1 day ago', () => {
+    const now = new Date('2017-08-12T12:00:00Z')
+    Date.now = jest.genMockFunction().mockReturnValue(now)
```

Comment:
Is this different to using `jest.fn()`?

Reply:
I've seen an example with "getMockFunction()" - but you're right: jest.fn() does the same and looks simpler.